plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixConnectorTest.java

```diff
@@ -307,6 +317,50 @@ public void testMissingColumnsOnInsert()
         assertQuery("SELECT * FROM test_col_insert", "SELECT 1, 'val1', 'val2'");
     }
 
+    @Override
+    public void testTopNPushdown()
+    {
+        assertThat(query("SELECT orderkey FROM orders ORDER BY orderkey LIMIT 10"))
+                .ordered()
+                .isNotFullyPushedDown(TopNNode.class, ExchangeNode.class, ExchangeNode.class, LimitNode.class);
+    }
+
+    @Test
+    public void testUseSortedProperties()
+    {
+        String tableName = "test_propagate_table_scan_sorting_properties";
+        String createTableSql = format("" +
+                        "CREATE TABLE %s AS " +
+                        "SELECT * FROM tpch.tiny.customer",
+                tableName);
+        assertUpdate(createTableSql, 1500L);
```

Comment:
Are multiple splits guaranteed in this test ?

Reply:
Good point. No, they are not.

Reply:
Added salting, which leads multiple splits.